src/components/NavBar /ShoppingCartItem.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+import SectionSeperator from "./SectionSeperator";
+
+const ShoppingCartItem = ({ planName, num1, num2 }) => {
+  const [hasCartItem, removeCartItem] = useState(true);
```

Comment:
the second parameter of useState should be prefixed with set followed by your first parameter name `setHasCartItem` as part of the general convention. It's easier to identify later on when you are looking at code to see which functions are state functions vs general logic functions